### PR TITLE
Remove API baselines for M.A.Testing

### PR DESCRIPTION
**Pull request by JunTaoLuo**
This is causing a bit of confusion for people. This is not a shipped package but was considered as an implementation project by our targets.

### CODE REVIEW

**Review by wtgodbe — APPROVED** [review]

**Comment by Pilchie** [comment]
But we do ship this for customers to use in their testing, right? So it *is* public, shipping API?

File: eng/ProjectReferences.props [hunk 1]
```diff
@@ -157,6 +157,5 @@
     <ProjectReferenceProvider Include="Microsoft.Extensions.WebEncoders" ProjectPath="$(RepoRoot)src\WebEncoders\src\Microsoft.Extensions.WebEncoders.csproj" />
     <ProjectReferenceProvider Include="Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions" ProjectPath="$(RepoRoot)src\HealthChecks\Abstractions\src\Microsoft.Extensions.Diagnostics.HealthChecks.Abstractions.csproj" />
     <ProjectReferenceProvider Include="Microsoft.Extensions.Diagnostics.HealthChecks" ProjectPath="$(RepoRoot)src\HealthChecks\HealthChecks\src\Microsoft.Extensions.Diagnostics.HealthChecks.csproj" />
-    <ProjectReferenceProvider Include="Microsoft.AspNetCore.Testing" ProjectPath="$(RepoRoot)src\Testing\src\Microsoft.AspNetCore.Testing.csproj" />
```
Comment (dougbu) [inline eng/ProjectReferences.props:160]: Undo this. The project is still a project reference provider and this change has nothing to do with the API baselines.

**Comment by BrennanConroy** [comment]
I think we only ship the `TestHost` which is under the hosting folder.

**Review by dougbu — COMMENTED** [review]
@BrennanConroy's correct @Pilchie. The M.A.Testing project is packable but non-shipping. I forget what other repos use it but the package will never appear on NuGet.org. (`<sarcasm>`Fond remembrance here of the release that included a specification tests package depending on the non-shipping Microsoft.AspNetCore.Testing package.`</sarcasm>`)

File: eng/ProjectReferences.props (hunk 1 above)
Comment (JunTaoLuo) [reply eng/ProjectReferences.props]: Yea, it turns out trusting codecheck without thinking is not a good idea.

**Pull request merged**